sgx/src/enclave/enclave.rs

```diff
@@ -141,41 +118,71 @@ impl Thread {
         Some(Self { enc, tcs, fnc })
     }
 
-    /// Issues `EENTER` instruction to the enclave.
+    /// Enter an enclave.
     ///
-    /// TODO add more comprehensive docs
+    /// This function enters an enclave using `Leaf` and provides the
```

Comment:
I think the reference to `Leaf` should be dropped here.

Reply:
Fixed.